README.md

```diff
@@ -12,6 +12,7 @@ Initial setup, installing dependencies using `pipenv`:
 
     $ pipenv install
     $ pipenv shell
+    $ pip install pytest-django
```

Comment:
If this is a new dependency, it should go in `Pipfile` so it is installed by pipenv. You should also edit the conda line to match.